garage/tf/algos/npo.py

```diff
@@ -1,23 +1,17 @@
 """Natural Policy Gradient Optimization."""
-from enum import Enum
-from enum import unique
+from enum import Enum, unique
 
 import numpy as np
 import tensorflow as tf
 
-from garage.misc import logger
+from garage.logger import HistogramInput, logger, tabular
 from garage.misc import special
 from garage.misc.overrides import overrides
 from garage.tf.algos import BatchPolopt
 from garage.tf.misc import tensor_utils
-from garage.tf.misc.tensor_utils import compute_advantages
-from garage.tf.misc.tensor_utils import discounted_returns
-from garage.tf.misc.tensor_utils import filter_valids
-from garage.tf.misc.tensor_utils import filter_valids_dict
-from garage.tf.misc.tensor_utils import flatten_batch
-from garage.tf.misc.tensor_utils import flatten_batch_dict
-from garage.tf.misc.tensor_utils import flatten_inputs
-from garage.tf.misc.tensor_utils import graph_inputs
+from garage.tf.misc.tensor_utils import (
```

Comment:
please don't replace the multi-line version unless the formatter is forcing you to